packages/server/test/unit/browsers/chrome_spec.js

```diff
@@ -228,6 +231,43 @@ describe('lib/browsers/chrome', () => {
       })
     })
 
+    it('install extension and ensure write access', function () {
+      mockfs({
```

Comment:
I think the reason this is failing is because `fs` is not being restored after the test. Try adding an `afterEach` that calls the callback returned by `mockfs` to reset the `fs`: https://github.com/tschaub/mock-fs#mockrestore

Reply:
Ah I see.. thanks for the hint @flotwig